Update introduction.pt.md (#1410)
mercadopago/devsite-docs: guides/plugins/woocommerce/introduction.en.md

# WooCommerce

> WARNING
>
> Protect your integration!
>

> > In October 2022, the WooCommerce plugin's versions before 4.5.0 will be discontinued. Keep your store always up to date with the latest version so you don't have to suspend transactions and expose sensitive data.

Reviewer: ```suggestion
> In October 2022, the WooCommerce plugin versions before 4.5.0 will be discontinued. Keep your store always up to date with the latest version to avoid having to suspend transactions and exposing sensitive data.
```
